### DISCUSSION TITLE
zh/guide/drivers/googlephotos

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#283] [Announcements]
# zh/guide/drivers/googlephotos

根文件夹 ID 根目录root，其他目录ID未知( 已解决下面有介绍) 获取 client_id, client_secret 获取工具 https://alist.nn.ci/tool/google/request 如何获取自己的 clientid(客户端ID) clientsecret(客户端秘钥) 去看谷歌网盘 (./googledrive.md...

https://alist.nn.ci/zh/guide/drivers/googlephotos.html

**Comment by svohoyzvozyvs** [comment]
可以删图片吗
